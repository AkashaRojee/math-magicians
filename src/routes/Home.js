/* eslint-disable */

const Home = () => {
  return (
    <span>home</span>
  );
}

export default Home;