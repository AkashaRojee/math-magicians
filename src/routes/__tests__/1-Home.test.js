import renderer from 'react-test-renderer';
import Home from '../1-Home';

test('Home route renders correctly', () => {
  const tree = renderer
    .create(<Home />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
