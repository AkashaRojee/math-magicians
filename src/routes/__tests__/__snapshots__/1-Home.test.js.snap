// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home route renders correctly 1`] = `
<main
  className="home-route"
>
  <h2>
    Welcome to our page!
  </h2>
  <div>
    <p>
      Lorem ipsum 1
    </p>
    <p>
      Lorem ipsum 2
    </p>
    <p>
      Lorem ipsum 3
    </p>
  </div>
</main>
`;
